contents/handbook/people/feedback.md

```diff
@@ -21,7 +21,7 @@ We run full team 360-degree feedback session as part of every offsite. The sessi
 
 - Preparation includes reading our handbook about how to be a good feedback [giver](/handbook/people/feedback#how-to-give-good-feedback) and [receiver](/handbook/people/feedback#how-to-receive-feedback-well).
 
-- As a guide the mix of positive and constructive feedback will vary. You should spend more time talking over the constructive even if you have a long list of positive things to share – this is an opportunity to help each other to grow. 
+- This is an opportunity to help each other to grow so include positive and constructive feedback. Constructive feedback is usually much more useful and much harder to come up with. Ideally you would share more constructive than positive feedback. If that's not possible you should spend more time talking over the constructive feedback - even if you have a long list of positive things to share.
```

Comment:
I personally prefer the other version. This now still feels like it is pushing me to come up with a big list of constructive feedback (maybe even things that aren't actually important or relevant) I dunno. Happy with either but my preference is for the other one.